core/package.json

```diff
@@ -30,7 +30,7 @@
     "loader/"
   ],
   "dependencies": {
-    "ionicons": "^4.6.3",
+    "ionicons": "^5.0.0-11",
```

Comment:
Should we be updating to a pre-release? I'd like to use a final `5.0` officially